daq/gcp.py

```diff
@@ -46,11 +46,9 @@ def __init__(self, config, callback_handler):
         self._client_name = self._parse_creds(cred_file)
         self._pubber = pubsub_v1.PublisherClient(credentials=self._credentials)
         LOGGER.info('Initialized gcp pub/sub %s:%s', self._project, self._client_name)
-        # TODO: Reinstate Firestore once it doesn't break Travis anymore.
-        # self._firestore = self._initialize_firestore(cred_file)
-        self._firestore = None
```

Comment:
Just curious if we still need the null check in "register_config".

Reply:
register_config is a public function and might be called in cases where _firestore is None (see code above for early exit), so it's still needed there.